bootbox.js

```diff
@@ -216,6 +216,24 @@
    * merge a set of default dialog options with user supplied arguments
    */
   function mergeArguments(defaults, args, properties) {
+    var mapArgumentsResult = mapArguments(
+      args,
+      properties
+    );
+    if(defaults && defaults.buttons && mapArgumentsResult && mapArgumentsResult.buttons){
```

Comment:
Oh sorry, you're right! Is there a way to auto fix the code, so I use the same standard. Maybe a linter or something like this. I suppose it should be an automatic (`grunt`?) task.